recipes/pyrser/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "pyrser" %}
+{% set version = "2.5.0" %}
```

Comment:
Where is this version coming from? PyPI has 0.2 and that's what I see in upstream?

Reply:
updated this to 0.2.0